openra/api.py

```diff
@@ -74,7 +74,7 @@ def __map_info_from_objects(request, map_objects, yaml):
         else:
             minimap = ''
 
-        download_url = 'http://' + request.META['HTTP_HOST'] + \
+        download_url = 'http://' + request.META.get('HTTP_HOST', 'resource.openra.net') + \
```

Comment:
Can you explain this change?

Reply:
It was giving an error in tests as the field wasn't set, so I put a default in to use when it's not. Maybe it'd be better coming from the config or something in the future though

Reply:
Ok on second thoughts I don't think this is needed, as the route that was tested didn't end up actually using that code in the end, and there's probably a better solution to hard coding a fallback. So I've reverted this now.